### PR TITLE
Enable dns stats by default

### PR DESCRIPTION
**Pull request by mbakht**
#### What this PR does / why we need it:

We have enabled DNS stats collection by default everywhere. Updating the helm chart to be in sync with that.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [ ] Chart Version bumped
- [ ] `CHANGELOG.md` has beed updated
- [ ] Variables are documented in the `README.md`

### CODE REVIEW

File: charts/datadog/values.yaml [hunk 1]
```diff
@@ -313,7 +313,7 @@ datadog:
     enableOOMKill: false
 
     # datadog.systemProbe.collectDNSStats -- Enable DNS stat collection
-    collectDNSStats: false
+    collectDNSStats: true
```
Comment (clamoriniere) [inline charts/datadog/values.yaml:316]: does it means that SystemProbe is required by default?

Comment (mbakht) [reply charts/datadog/values.yaml:316]: It is the latter. DNS stats will get collected only when `networkMonitoring.enable` is also `true`

Comment (clamoriniere) [reply charts/datadog/values.yaml:316]: Thanks for the answer. 
If you can rebase you pull request, we will be able to merge it.

Comment (mbakht) [reply charts/datadog/values.yaml:322]: Done

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**